ReactCommon/jsi/jsi/jsi.h

```diff
@@ -937,7 +937,9 @@ class JSI_EXPORT Value {
             std::is_base_of<String, T>::value ||
             std::is_base_of<Object, T>::value,
         "Value cannot be implicitly move-constructed from this type");
+#ifndef __clang_analyzer__ // Disable [bugprone-move-forwarding-reference] when running clang static analysis
```

Comment:
Can you add a `// TODO(macOS GH#774)` comment around this so it's obvious this is a purposeful change _we_ made in our fork.

Reply:
Added in latest commit